src/main/scala/com/atomist/rug/runtime/rugdsl/RugOperationSupport.scala

```diff
@@ -157,9 +157,12 @@ trait RugOperationSupport extends LazyLogging {
                         context: TreeNode,
                         poa: ProjectOperationArguments,
                         identifierMap: Map[String, Object]): Seq[TreeNode] = {
-    val vo = DefaultViewFinder.findIn(rugAs, selected, context, poa, identifierMap)
-    vo.getOrElse {
-      throw new RugRuntimeException(null, s"Cannot find type '${selected.kind}' under $context using $DefaultViewFinder")
+    DefaultViewFinder.findIn(rugAs, selected, context, poa, identifierMap) match {
+      case Left(s) if s.isEmpty =>
+        throw new RugRuntimeException(null, s"Cannot find '${selected.kind}' under $context. It has no children of its own.")
```

Comment:
I wonder if you couldn't create a new exception type for this? Or use one that only takes a string?